### ISSUE TITLE
NullPointerException in PreviewService

### ISSUE THREAD

**Issue by dzeikei** [intercom/intercom-android#102]
```
java.lang.RuntimeException: Unable to start service io.intercom.android.sdk.preview.PreviewService@141ff845 with Intent { cmp=com.cammy.cammy.dev/io.intercom.android.sdk.preview.PreviewService }: java.lang.NullPointerException: Attempt to invoke interface method 'boolean java.util.List.isEmpty()' on a null object reference
            at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3076)
            at android.app.ActivityThread.access$2100(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1465)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5499)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:737)
     Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'boolean java.util.List.isEmpty()' on a null object reference
            at io.intercom.android.sdk.preview.PreviewService.checkConversations(PreviewService.java:263)
            at io.intercom.android.sdk.preview.PreviewService.onStartCommand(PreviewService.java:157)
            at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3037)
            at android.app.ActivityThread.access$2100(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1465)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5499)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:737)
```

**Comment by dzeikei** [comment]
Version 1.0.4

**Comment by harkin** [comment]
Thanks for letting us know about this one, we've already fixed the underlying issue and that fix will be in v1.0.5

**Comment by harkin** [comment]
Going to close this issue in favour of https://github.com/intercom/intercom-android/issues/95

There's more discussion on that one
